views/error.ejs

```diff
@@ -0,0 +1,3 @@
+<h1><%= message %></h1>
+<h2><%= error.status %></h2>
```

Comment:
So this is how I can comment on any line of The added code or suggest some changes.